### PR TITLE
[QnA] Fix QnA Unit Tests

### PR DESCRIPTION
**Pull request by Batta32**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

**Review by matiasroldan6 — APPROVED** [review]

File: libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/testData/QnaMaker_ReturnsAnswer_GivenScoreThresholdQueryOption.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "answers": [
         {
             "score": 68.54820341616869,
-            "Id": 22,
+            "id": 22,
```
Comment (Batta32) [inline libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/testData/QnaMaker_ReturnsAnswer_GivenScoreThresholdQueryOption.json:5]: This is the only change that I'm not sure about

**Review by fran893 — CHANGES_REQUESTED** [review]
We need to remove the comments `Extract V3 response` and `Get Oracle file` that are present in the QnAMakerTests and QnAMakerRecognizersTests classes, I think they are wrong

File: libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/testData/QnaMaker_ReturnsAnswer_GivenScoreThresholdQueryOption.json (hunk 1 above)
Comment (fran893) [reply libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/testData/QnaMaker_ReturnsAnswer_GivenScoreThresholdQueryOption.json:5]: Agree, anyway, if we left it as the original version we need to find a workaround for the test that use this file

**Comment by Batta32** [comment]
Agree with you

File: libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/testData/QnaMaker_ReturnsAnswer_GivenScoreThresholdQueryOption.json (hunk 1 above)
Comment (Batta32) [reply libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/testData/QnaMaker_ReturnsAnswer_GivenScoreThresholdQueryOption.json:5]: I executed the tests, but it passed correctly.. I'll double check.

**Review by FedericoBernal — APPROVED** [review]
LGTM

File: libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/testData/QnaMaker_ReturnsAnswer_GivenScoreThresholdQueryOption.json (hunk 1 above)
Comment (fran893) [reply libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/testData/QnaMaker_ReturnsAnswer_GivenScoreThresholdQueryOption.json:5]: Yes, they are passing rollbacking the change, it's weird, when I tested it did not pass with the original file

Comment (Batta32) [reply libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/testData/QnaMaker_ReturnsAnswer_GivenScoreThresholdQueryOption.json:5]: I'll confirm this and I'll rollback the change.. if not.. this should be a problem on your side as we are not in parity

Comment (Batta32) [reply libraries/bot-ai-qna/src/test/java/com/microsoft/bot/ai/qna/testData/QnaMaker_ReturnsAnswer_GivenScoreThresholdQueryOption.json:5]: Fixed

**Review by fran893 — APPROVED** [review]

**Pull request merged**